components/Pagination/sub-components/ActionButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Colors from '../../Colors';
+
+const StyledButton = styled.a`
+  border: 1px solid transparent;
+  border-radius: 10px;
+  cursor: pointer;
+  font-weight: normal;
+  margin: 0 4px;
+  outline: none;
+  padding: 8px;
+  text-align: center;
+  text-decoration: none;
+  transition: color 0.2s ease-in-out, background-color 0.2s ease-in-out,
```

Comment:
Do we have transition/animation definitions?

Reply:
No.

Reply:
So, why those transitions?

Reply:
We have transition in several components, like `Button`, `Checkbox`, `Tooltip`, etc.
It's just to be a little bit smooth, what do you think?

Or should we remove the transitions from all components, since we do not have transition/animations definition to none of them?

Reply:
Really don't know. 

We just need to be cautious since we add this by our own. IMO, it could be added to all components but, we always need to keep the UX team in alignment with this.

I don't think we should remove the transitions from all components. Instead, we should avoid adding it to the newers.

Reply:
It was aligned with the UX team, he made his review of how the usability is, and, for him, it's okay.

As next steps I think we need to get this definitions of animations documented by the UX team to get this centralized and make reusable through all components.